tests/reporter/kubernetes.go

```diff
@@ -171,6 +171,11 @@ func (r *KubernetesReporter) dumpNamespaces(duration time.Duration, vmiNamespace
 	r.logVMExports(virtCli)
 	r.logDeployments(virtCli)
 	r.logDaemonsets(virtCli)
+	r.logVolumeSnapshots(virtCli)
+	r.logVolumeSnapshotContents(virtCli)
+	r.logVolumeSnapshotClasses(virtCli)
+	r.logVirtualMachineSnapshots(virtCli)
+	r.logVirtualMachineSnapshotContents(virtCli)
```

Comment:
Can we add DataVolumes to the list?

Reply:
https://github.com/kubevirt/kubevirt/blob/main/tests/reporter/kubernetes.go#L170 should already be there.